src/commands/pools/show_map.ts

```diff
@@ -0,0 +1,33 @@
+import {ApplicationCommandOptionTypes} from "discord.js/typings/enums";
+import {ICommand} from "wokcommands";
+import {showMap} from "../../abstract_commands/pools/show";
+import {ensure} from "../../utils/general";
+
+export default {
+    category: "General",
+    description: "Show statics of a map",
+    slash: true,
+    testOnly: true,
+    guildOnly: true,
+
+    options: [
+        {
+            name: "name",
+            description: "The name of the map",
```

Comment:
Please use UUID
`            description: "The uuid of the map to show",`

Reply:
Resolved.